pkg/oomstore/apply.go

```diff
@@ -196,15 +196,31 @@ func buildApplyStage(ctx context.Context, opt apply.ApplyOpt) (*apply.ApplyStage
 				return nil, err
 			}
 
-			status.NewEntities = append(status.NewEntities, entity)
+			stage.NewEntities = append(stage.NewEntities, apply.Entity{
```

Comment:
can you explain why?

Reply:
We don't want `entity.BatchFeature` to have values. The whole stage should be a flat structure, not a nested structure.

Reply:
can you add a comment in code? so that people don't ask

Reply:
sound good, it's done.